tokio-trace-fmt/src/lib.rs

```diff
@@ -216,7 +247,7 @@ impl Default for Builder {
 impl<N, E, F> Builder<N, E, F>
```

Comment:
WDYT about adding methods to `Builder<N, default::Format, F>` for configuring the timestamps?